packages/amplify-graphql-api-construct-tests/src/commands.ts

```diff
@@ -0,0 +1,55 @@
+import * as path from 'path';
+import * as fs from 'fs';
+import { copySync, moveSync, readFileSync } from 'fs-extra';
+import { getScriptRunnerPath, nspawn as spawn } from 'amplify-category-api-e2e-core';
+
+const getNpxPath = (): string => (process.platform === 'win32' ? getScriptRunnerPath().replace('node.exe', 'npx.cmd') : 'npx');
```

Comment:
Note: all methods in this class are migrated from the e2e-core package (and removed from that package), then just refactored to align w/ our current style guidelines.

The only notable change that should speed things up a bit is that I now install all deps and devdeps in parallel during setup.